README.md

```diff
@@ -17,27 +18,28 @@ In addition, each Section can have its state(Loading/Loaded/Failed) controlled i
 
 ## Gradle Dependency
 
+Add this to the `dependencies` section in your project-level **build.gradle** file:
+
 ```groovy
-dependencies {
-    compile 'io.github.luizgrp.sectionedrecyclerviewadapter:sectionedrecyclerviewadapter:1.0.5'
-}
+compile 'io.github.luizgrp.sectionedrecyclerviewadapter:sectionedrecyclerviewadapter:1.0.5'
 ```
 
-## Basic usage:
+## Basic usage
+
 ##### 1) Create a custom Section class extending StatelessSection
+
 ```java
 class MySection extends StatelessSection {
-
-    List<String> myList = Arrays.asList("Item1", "Item2", "Item3");
+    List<String> mItemList = Arrays.asList("Item1", "Item2", "Item3");
 
     public MySection() {
-        // call constructor with layout resources for this Section header and items 
+        // call constructor with layout resources for this Section header and items
         super(R.layout.section_header, R.layout.section_item);
     }
 
     @Override
     public int getContentItemsTotal() {
-        return myList.size(); // number of items of this section
+        return mItemList.size(); // number of items of this section
```

Comment:
Even though the `m` prefix looks very similar to Hungarian notation, it actually identifies the *scope*, not the *type* of the variable, as defined in the [Field Naming Conventions](https://source.android.com/source/code-style#follow-field-naming-conventions) of the AOSP style guide.
But if you prefer variable names without the prefix, that's fine too ;) I removed the prefixes in my updated PR.